src/client/pps/pps.proto

```diff
@@ -116,6 +123,7 @@ message JobInfo {
   uint64 restart = 20;
   int64 data_processed = 22;
   int64 data_total = 23;
+  repeated WorkerStatus worker_status = 24;
```

Comment:
You are right observing this, although it's not a super strong preference. Basically at some point Peter started doing this and I kept doing it for consistency sake which I feel more strongly about than about which option we pick. There are a few things I don't like about plurals, they're irregular which can make it hard to grep for them, `worker_status` happens to be a substring of `worker_statuses` but the same is not true of `datum` and `data` for example.